src/IssueViz.Security/HotspotsList/TableDataSource/HotspotsTableColumns.cs

```diff
@@ -29,8 +29,9 @@ internal static class HotspotsTableColumns
     {
         public static IReadOnlyList<ColumnState> InitialStates { get; } = new[]
         {
-            new ColumnState(StandardTableColumnDefinitions.ErrorCode, true, 50),
-            new ColumnState(PriorityTableColumnDefinition.ColumnName, true, 100),
+            new ColumnState(NavigabilityTableColumnDefinition.ColumnName, true, NavigabilityTableColumnDefinition.Width),
```

Comment:
No copy-paste of the width

Reply:
I don't understand your comment.

Reply:
The width value was copy-pasted: here and in the column definition. So now I simply reference the value in the column definition.